app.config.js

```diff
@@ -74,5 +74,29 @@ module.exports = {
   // is used to create and show a privacy preference center / cookie banner
   // To learn more about how to configure and use this, please refer to the readme
   privacyPreferenceCenter:
-    process.env.GATSBY_PRIVACY_PREFERENCE_CENTER || 'false'
+    process.env.GATSBY_PRIVACY_PREFERENCE_CENTER || 'false',
+
+  // Migration contract addresses
+  ethereumMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_ETHEREUM || '',
+  polygonMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_POLYGON || '',
+  bscMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_BSC || '',
+  moonriverMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_MOONRIVER || '',
+  ewcMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_EWC || '',
+  rinkebyMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_RINKEBY ||
+    '0x0e4806e14bd02A809f49aD37F8C749f117D84239',
+  ropstenMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_RINKEBY ||
+    '0x003323388790872320fbf0152a49E2de4046E0b3',
+  moonbaseMigrationContractAddresss:
+    process.env.GATSBY_MIGRATION_CONTRACT_ADDRESS_RINKEBY ||
+    '0x7874E8bA0f96fA3cD883453a596BF97aEd5c7a32',
+  mumbaiMigrationContractAddresss:
```

Comment:
this is full of networks we never intended to do a "migration" for so please clean up